app/Dockerfile

```diff
@@ -1,34 +1,42 @@
 FROM python:3.8.2
 
+WORKDIR /app
+EXPOSE 7001
+
+
 # Disable python pyc files
 ENV PYTHONDONTWRITEBYTECODE 1
 
-WORKDIR /app
-
-EXPOSE 7001
 
 # Script for waiting for the db to initialize
 COPY bin/wait-for-it.sh /app/bin/wait-for-it.sh
 RUN chmod +x /app/bin/wait-for-it.sh
 
+
+# Install nvm with node and npm
+ENV NODE_VERSION=14.8.0
```

Comment:
is this gonna be a temp thing? specifying the exact node version till the issue is fixed i mean.

Reply:
Yeah I hope so, we can probably revert this whole pr when it gets fixed, maybe we'll give it a couple weeks.

Reply:
:+1:&nbsp; I filed #3502 to investigate reverting this in the future.